src/components/menu/MenuItem.tsx

```diff
@@ -46,9 +17,15 @@ export const MenuItem = ({
   className,
   icon,
   onClick,
+  style,
 }: MenuItemProps) => (
-  <StyledListItem className={className} onClick={onClick}>
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
```

Comment:
We may want to disable the `jsx-a11y/click-events-have-key-events` rules since we don't always have the key events for each click event now. We might want to in the future, but I don't think now is the time, especially since we are migrating to `css-modules`.